frontend/utils/formatTime.ts

```diff
@@ -0,0 +1,5 @@
+import dayjs from "dayjs";
+
+export function formatDateTime(date: string) {
```

Comment:
could be nice if the format is customisable. so it can be reusable

Reply:
Done